docs/api/plugins.md

```diff
@@ -153,6 +153,13 @@ event, e.g., `server.on('after', plugins.metrics());`:
   * `res` {Object} the response obj
   * `route` {Object} the route obj that serviced the request
 
+The module includes the following plugins to be used with restify's `pre` event:
+* `inflightRequestThrottle(options)` - limits the max number of inflight requests
+  * `options.limit` {Number} the maximum number of inflight requests the server will handle before returning an error
+  * `options.res` {Error} An error that will be passed to `res.send` when the limit is reached.
```

Comment:
Yup, I totally agree. This grew organically since the original implementation expected this to be an array of parameters that would be passed to `res.send.apply`. Now that we expect a restify error, its an awkward fit.